index.js

```diff
@@ -75,119 +71,114 @@ var PersonaClient = function (config) {
 /**
  * Validate bearer token against cache or persona
  * @param token: Token to validate
- * @param scope: Requested scope
- * @param overrideScope: Backup scope to use instead of scope
+ * @param scopes: An array of one or more scopes that the token must have to be valid.
  * @param next: Callback, function(err, validatedBy)
  */
-PersonaClient.prototype.validateToken = function (token, scope, overridingScope, next) {
+PersonaClient.prototype.validateToken = function (token, scopes, next) {
     if (!next) {
         throw "No callback (next attribute) provided";
     }
 
     if (token == null) {
-        next(ERROR_TYPES.INVALID_TOKEN, null);
-        return;
+        return next(ERROR_TYPES.INVALID_TOKEN, null);
     }
 
-    var _this = this,
-        cacheKey = token + ((scope) ? "@" + scope : "");
-    // if we were given a scope then append the scope to the token to create a cachekey
-
-    this.debug("Validating token: " + cacheKey);
-    this.redisClient.get("access_token:" + cacheKey, function (err, reply) {
-        if (reply === "OK") {
-            _this.debug("Token " + cacheKey + " verified by cache");
-            next(null, VALIDATED_TYPES.CACHE);
-        } else {
-            var requestPath = _this.config.persona_oauth_route + token;
-            var usingScope = scope || overridingScope || null;
+    if (_.isEmpty(scopes)) {
+        return next(ERROR_TYPES.INSUFFICIENT_SCOPE, null);
+    }
+
+    this.debug("Validating token: " + token);
+    var _this = this;
```

Comment:
I'll add the lint rules to the project and run it in warning mode. You must be getting tired of being a human linter ;)